### PR TITLE
Automated testing

### PR DESCRIPTION
**Pull request by EnisBourbia**
Resolves: #53 

Test cases for automated testing added for the client application.

### CODE REVIEW

File: front_end/package-lock.json [hunk 1]
```diff
@@ -21,6 +21,12 @@
         "sumchecker": "^3.0.1"
       }
     },
```
Comment (VladLiteanu) [inline front_end/package-lock.json:23]: We probably don't need to update this file as it gets updated automatically when we do "npm install".

File: front_end/test/test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+describe("Application launch test", function () {
+  this.timeout(20000)
+	
+	beforeEach(function () {
+		
+    const appPath = path.join(__dirname, "../");
+
+		  this.app = new Application({
```
Comment (VladLiteanu) [inline front_end/test/test.js:15]: This file could be formatted better to be more consistent with the rest of the code.

**Review by VladLiteanu — COMMENTED** [review]
Very good job overall!

File: front_end/package-lock.json (hunk 1 above)
Comment (EnisBourbia) [reply front_end/package-lock.json:23]: It seems that the package-lock is required for the tests to run properly.

**Review by VladLiteanu — APPROVED** [review]
Very good job!

**Pull request merged**